docs/sphinx/sections/api/apidocs/pipes.rst

```diff
@@ -1,32 +1,60 @@
+Dagster Pipes
+=============
+
 .. currentmodule:: dagster
 
-Pipes
-=====
+`Dagster Pipes </concepts/dagster-pipes>`_  is a toolkit for building integrations between Dagster and external execution environments. This reference outlines the APIs included with the ``dagster`` library, which should be used in the orchestration environment.
+
+For a detailed look at the Pipes process, including how to customize it, refer to the `Dagster Pipes details and customization guide </concepts/dagster-pipes/dagster-pipes-details-and-customization#overview-and-terms>`_.
+
+**Looking to write code in an external process?** Refer to the API reference for the separately-installed `dagster-pipes </_apidocs/libraries/dagster-pipes>`_ library. 
+
+----
 
-Abstractions for the orchestration side of the Dagster Pipes protocol.
+Sessions
+--------
+
+.. autoclass:: PipesSession
+
+.. autofunction:: open_pipes_session
+
+----
+
+Clients
+-------
 
 .. autoclass:: PipesClient
 
 .. autoclass:: PipesSubprocessClient
 
-.. autoclass:: PipesContextInjector
+----
 
-.. autoclass:: PipesMessageReader
+Context injectors
```

Comment:
Same commentary here. This and below are largely for people customizing the protocol.